python/vineyard/__init__.py

```diff
@@ -50,7 +50,8 @@ def _import_module_from_file(filepath):
                 spec = importlib.util.spec_from_file_location("vineyard._contrib", filepath)
                 mod = importlib.util.module_from_spec(spec)
                 spec.loader.exec_module(mod)
-            except:
+            except Exception as e:
+                print(e)
                 pass
```

Comment:
Delete those debug lines, and you shouldn't touch `vineyard/__init__.py`, according to the driver's design, the stream.py, or something that imports stream.py, should be placed under any of the following directories:

* /etc/vineyard/config.py
* {sys.prefix}/etc/vineyard/config.py
* /usr/local/share/vineyard/01-xxx.py
* {sys.prefix}/share/vineyard/02-xxxx.py
* $HOME/.vineyard/03-xxxxx.py